File: web/MongoProyectoWeb/MongoProyectoWeb/Models/ReservasModel.cs
using System.Text.Json.Serialization;

namespace MongoProyectoWeb.Models
{
    public class ReservasModel
    {
        [JsonPropertyName("_id")]
        public int? _id { get; set; }

        [JsonPropertyName("id_usuario")]
        public int IdUsuario { get; set; }

        [JsonPropertyName("id_local")]
        public int IdLocal { get; set; }

        [JsonPropertyName("fecha")]
        public string fecha { get; set; }

        [JsonPropertyName("hora")]
        public string hora { get; set; }

        [JsonPropertyName("numero_personas")]
        public int numero_personas { get; set; }

        [JsonPropertyName("descripcion")]
        public string descripcion { get; set; }
    }
}

File: api/src/models/ReservasModel.js
const mongoose = require('mongoose');

const ReservasSchema = new mongoose.Schema({
  _id: { type: Number, required: true },  
  id_usuario: { type: Number, required: true },  
  id_local: { type: Number, required: true },  
  fecha: { type: String, required: true },  
  hora: { type: String, required: true },
  numero_personas: { type: Number, required: true },
  descripcion: { type: String, required: true }
}, { collection: 'Reservas' });

module.exports = mongoose.model('Reservas', ReservasSchema);
